### PR TITLE
Add MSVC Component Llvm ClangToolset to VM

### PR DESCRIPTION
**Pull request by MVoz**
> https://github.com/reupen/azure-pipelines-image-generation/blob/master/images/win/scripts/Installers/Vs2019/Install-VS2019.ps1
> 
> https://github.com/microsoft/azure-pipelines-image-generation/pull/1297/files

### CODE REVIEW

**Comment by Voskrese** [comment]
`argument(s): "The directory is not empty.`

**error** (x64_uwp) OR (arm_uwp) !?

**Comment by Voskrese** [comment]
`<LLVMInstallDir>PATH_TO_LLVM</LLVMInstallDir>`

`msbuild /p:LLVMInstallDir="C:\Program Files (x86)\LLVM" /p:PlatformToolset="ClangCL"`

File: scripts/azure-pipelines/windows/provision-image.ps1 [hunk 1]
```diff
@@ -429,7 +438,7 @@ Add-MPPreference -ExclusionProcess cl.exe
 Add-MPPreference -ExclusionProcess link.exe
 Add-MPPreference -ExclusionProcess python.exe
 
-InstallVisualStudio -Workloads $Workloads -BootstrapperUrl $VisualStudioBootstrapperUrl -Nickname 'Stable'
+InstallVisualStudio -includeRecommended -Workloads $Workloads -BootstrapperUrl $VisualStudioBootstrapperUrl -Nickname 'Stable'
```
Comment (BillyONeal) [inline scripts/azure-pipelines/windows/provision-image.ps1:441]: This creates the liklihood of the installed bits changing a lot over time, so I don't think we should do that.

**Review by BillyONeal — CHANGES_REQUESTED** [review]
What is the use case you intend to enable with this?

**Comment by Voskrese** [comment]
@BillyONeal
https://github.com/microsoft/vcpkg/pull/11140

**Comment by Voskrese** [comment]
😕

**Comment by Voskrese** [comment]
the project apparently does not want to develop

**Comment by BillyONeal** [comment]
@Voskrese It isn't that 'we don't want to develop' -- I've been blocked from updating the VMs due to an unrelated issue. That's a heavy switch to flip since we need to get a clean baseline run when we do that. I've applied the install of LLVM over here: https://github.com/microsoft/vcpkg/pull/11365

As for 'are we going to add that as a triplet', that's a different question. Given that we ship LLVM in the box though it might make sense. /cc @ras0219-msft @ras0219

**Comment by Voskrese** [comment]
ok

I needed only `Microsoft.VisualStudio.Component.VC.Llvm.ClangToolset`

but without the dependencies of Microsoft.VisualStudio.Component.VC.Llvm.Clang it won't install

**Comment by BillyONeal** [comment]
> but without the dependencies of `Microsoft.VisualStudio.Component.VC.Llvm.Clang` it won't install

I observe the installer adding the right dependencies; I did this testing with 16.6. Rest assured when I get a VM update landed I'll make sure it's on the box

**Comment by Voskrese** [comment]
okey

**Comment by BillyONeal** [comment]
@Voskrese I believe your VM changes landed in https://github.com/microsoft/vcpkg/pull/11365

**Pull request closed without merging**